app/experimenter/nimbus-ui/src/components/PageResults/index.tsx

```diff
@@ -221,6 +246,7 @@ const PageResults: React.FunctionComponent<RouteComponentProps> = () => {
 
         <div>
           <h2 className="h4 mb-3">Outcome Metrics</h2>
+          {controlBranchError && controlBranchError}
```

Comment:
Is this a trick to show if it's not null or hide it otherwise?

Reply:
Yes. Might be a better way, but I tested and don't think there are any issues with doing it this way.

Reply:
I feel like I remember cases where these evaluate to `null` or `undefined` will prevent them from being displayed, I may be wrong on that though. Not sure that this is a more clear solution though if it does work

```suggestion
          {controlBranchError || null}
```

Reply:
Yep, this seems right to me too, I knew there was probably a better way thanks all.